Microsoft.Toolkit.Uwp.UI.Controls/Expander/Expander.cs

```diff
@@ -40,6 +41,33 @@ protected override void OnApplyTemplate()
             {
                 VisualStateManager.GoToState(this, StateContentExpanded, false);
             }
+
+            var button = (ToggleButton)GetTemplateChild(ExpanderToggleButtonPart);
+
+            if (button != null)
+            {
+                button.KeyDown -= ExpanderToggleButtonPart_KeyDown;
+                button.KeyDown += ExpanderToggleButtonPart_KeyDown;
+            }
+        }
+
+        private void ExpanderToggleButtonPart_KeyDown(object sender, Windows.UI.Xaml.Input.KeyRoutedEventArgs e)
+        {
```

Comment:
https://github.com/Microsoft/UWPCommunityToolkit/blob/dev/contributing.md#quality-insurance-for-pull-requests-for-xaml-controls ==> "Action must be triggered when hitting Enter key"